include/mbedtls/ccm.h

```diff
@@ -32,13 +32,11 @@
 extern "C" {
 #endif
 
-/**
- * \brief          CCM context structure
- */
-typedef struct {
-    mbedtls_cipher_context_t cipher_ctx;    /*!< cipher context used */
-}
-mbedtls_ccm_context;
+#if defined (MBEDTLS_CCM_ALT)
+#include "ccm_alt.h"
+#else
+typedef mbedtls_ccm_context_defualt mbedtls_ccm_context;
```

Comment:
Typo (here and in `ccm_internal.h`):  `mbedtls_ccm_context_defualt` -> `mbedtls_ccm_context_default`.